README.md

```diff
@@ -1,3 +1,61 @@
 # Libs
 
-A collection of shared libraries for use cross project at The Guardian
+> A collection of shared JavaScript libraries for use in Guardian projects
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
```

Comment:
Does this need to happen or is it now in CI?

Reply:
yeah happens in https://github.com/guardian/libs/blob/de953f00df5d44c205e5c80913896d81242fdabe/.github/workflows/CI.yml#L7-L16